crates/sui-core/src/consensus_adapter.rs

```diff
@@ -671,14 +671,15 @@ impl CheckpointConsensusAdapter {
                     let other = fragment.other.auth_signature.authority;
                     let transaction = ConsensusTransaction::new_checkpoint_message(fragment);
                     let tracking_id = transaction.get_tracking_id();
+                    let serialized = bincode::serialize(&transaction).expect("Serialize consensus transaction cannot fail");
                     debug!(
                         ?tracking_id,
                         ?cp_seq,
+                        size=?serialized.len(),
                         "Checkpoint fragment consensus message created. Proposer: {}, Other: {}",
                         proposer,
```

Comment:
These could be put into fields too.

Reply:
What do you mean?

Reply:
Sorry I mean that you could move `proposer` and the Other field up before the text so they are key=value pairs as well.... instead of using `Proposer: {}   Other: {}`

Reply:
Oh they are different though. With `{}` it will trigger the `Display` implementation. While putting in the fields would be equivalent to `{:?}` which is not desired in this case.

Reply:
It's not true that fields is equivalent to `{:?}`.  That is true only if you put `?var` or `key =? var`...  if you do just `var` or `key=var` then it uses `Display` or `Value`.   However if it is `Value` then one has to implement `Value` for our types which is a bit of a pain I guess.